docs/source/API/core/policies/RangePolicy.rst

```diff
@@ -11,60 +11,66 @@ Usage
 
```

Comment:
Please add a 1 - 3 sentence human-readable description of what RangePolicy is, and what it does.